docs/changes.rst

```diff
@@ -45,6 +45,12 @@ Released: not yet
 * Added the 'python_requires' keyword to the package definition, which makes pip
   aware of the supported Python versions.
 
+* Comment out those methods in the TupleParser that are just WBEM server parser
+  methods including iparamvalue, etc. They should never be received by the
+  pywbem client. We left the code in case the server is implemented but this
+  was better than adding the test for these methods. Add test for is_subclass
+  function in cim_operations.py
+
```

Comment:
I disagree with commenting out the server-related parse functions. We don't know whether somebody used pywbem to build a server. This kind if thing should be done in a new major version, i.e. in 1.0.0.

Reply:
In retrospect I agree.  I am going to modify to add tests for these functions.
I agree also with cim_http, in particular since is is really better covered than the numbers but part is with py27 and rest with py3 tests.

The one place we could gain probably easily is the compiler I would like to try to clean some of those up this week.

Reply:
DONE: Restored then and added tests for them.

Reply:
I forget to update the PR documentation.  Am doing that.